spec/unit/pushprocessor.spec.ts

```diff
@@ -819,21 +819,21 @@ describe("rewriteDefaultRules", () => {
 
         // By the time we get here, we expect the PushProcessor to have merged the new .m.rule.is_room_mention rule into the existing list of rules.
         // Check that has happened, and that it is in the right place.
-        const containsDisplayNameRuleIdx = pushRules.global.override?.findIndex(
+        const containsDisplayNameRuleIdx = pushRules.globalThis.override?.findIndex(
```

Comment:
🙃

Reply:
Believe it or not I did actually do these replaces manually, and still fucked it up :grimacing: